.gitignore

```diff
@@ -42,4 +43,10 @@ inventories_private/
 /*.xml
 
 # VM folder
-vm_images/*.qcow2
\ No newline at end of file
+vm_images/*.qcow2
+
+# Src from privates repos
+src/trafgen/*
+!src/trafgen/README
+src/svtools/*
+!src/svtools/README
```

Comment:
Add one empty line at the end of the file (good practice)
It helps compatibility with many tools using cat for example.